runtime/turing/src/lib.rs

```diff
@@ -373,6 +373,7 @@ parameter_type_with_key! {
 	pub ExistentialDeposits: |currency_id: CurrencyId| -> Balance {
 		match currency_id {
 			CurrencyId::Native => EXISTENTIAL_DEPOSIT,
+			CurrencyId::UNIT => 10 * CurrencyId::UNIT.millicent(),
```

Comment:
Do we need this token for Turing?

Reply:
hm i recall we were deprecating neumann and only keeping it around so that it's easier for us to restructure our code with multiple chain feature flags once we get OAK. so i've mainly been doing development on turing-dev. so with that, i imagined that it would probably be good to have this currency be there as well. would that be fine or should be just keep it contained to neumann for now?